.docs/10-custom-encodings.md

```diff
@@ -0,0 +1,11 @@
+[← Configuring Transports][back] - [:book:][index] - [Custom Transports →][next]
+
+# Custom Encodings
```

Comment:
Packages like protobuf and thrift we call encodings, but they might be better called IDL models, which may provide one or more codexes/encodings/wire protocols.